i18n/locales/en.ts

```diff
@@ -1507,20 +1507,36 @@ const translations = {
   chapter_seven: {
     title: `Offense is the best defense`,
     paragraph_one:
-      'It’s late, and you’re getting sleepy, but as you shut your eyes for a moment, a double dose of bad news arrives. 1) Vanderpoole is back on TV. 2) He is claiming to be the great-grandson of Satoshi Nakamoto. He also looks sleep-deprived.',
+      'You and Mike Ramen arrive first via plane, then via dune buggy, and then via rowboat at Vanderpoole’s private island.',
     intro_one: {
-      title: 'Building blocks',
+      title: 'Intro',
+      paragraph_one:
+        'You ask your compatriots whether they are worried about being spotted by Vanderpoole’s army of security drones.',
+      paragraph_two:
+        '—HOLOCAT: “Give me 15 minutes, and I’ll take care of them. They’re birds, and I’m a cat. They don’t stand a chance.”',
+      paragraph_three:
+        'Holocat extends his claws and deals with Vanderpoole’s drone army. Mike Ramen hands you a black turtleneck sweater, gloves, and a pair of night vision goggles.',
```

Comment:
missing. a "-"  from the design

Reply:
ahh yeah, I was referencing the updated designs with the latest images and the "-" was removed